src/control/server/storage/bdev/backend_vmd.go

```diff
@@ -23,63 +23,78 @@ import (
 
 // backingAddrToVMD converts a VMD backing devices address e.g. 5d0505:03.00.0
 // to the relevant logical VMD address e.g. 0000:5d:05.5.
-func backingAddrToVMD(backingAddr string) (string, bool) {
-	domain, _, _, _, err := common.ParsePCIAddress(backingAddr)
-	if err != nil {
-		return "", false
-	}
-	if domain == 0 {
-		return "", false
+func backingAddrToVMD(in common.PCIAddress) (common.PCIAddress, bool) {
```

Comment:
Having this logic in two places, I think it should be centralized. Whether that's in pci_utils.go or exporting the lib/spdk package functions.